### ISSUE TITLE
File must begin with "/": %{rpmmacrodir}/macros.httpd

### ISSUE THREAD

**Issue by twistedatrocity** [iuscommunity-pkg/httpd24u#25]
Trying to build src rpm (httpd24u-2.4.37-1.ius.centos6.src.rpm) on CentOS 6.10 x86_64 box using rpmbuild -bb and it fails here:

error: File must begin with "/": %{rpmmacrodir}/macros.httpd
RPM build errors:
    File must begin with "/": %{rpmmacrodir}/macros.httpd

I have only edited the spec file to use 2.4.38 source tarball and --with-ssl=/usr/local/openssl in the configure options (so that h2 will actually work since it needs 1.0.2 and there is no rpm for this version of openssl)

**Comment by twistedatrocity** [comment]
I found the issue. I needed to install the package "epel-rpm-macros"

**Comment by carlwgeorge** [comment]
Yup.  Just like ius-release depends on epel-release, many IUS spec files utilize epel macros.  On a system with EPEL enabled, `yum install @buildsys-build` will pull in epel-rpm-macros by default.  That is the same command mock uses to bootstrap it's chroots.

**Comment by carlwgeorge** [comment]
Feel free to open a pull request to this repo with your 2.4.38 changes!  I've been aware it was released, I just haven't found the time to look at it yet.

**Comment by twistedatrocity** [comment]
Thanks Carl. I usually don't do group based installs with yum hence I missed the package. and of course the bleeding edge rpm's have their own version of rpmmacros now too which made it more confusing haha

I do not have a git checkout of the spec, however I may do this if it helps. I did add epel-rpm-macros to the BuildRequires section, maybe that would be good to have for folks like me that pull the src and may not know better at first LOL

The 2.4.38 I made no changes to the apache source itself, no patches

Mainly it has to compile against at least openssl1.0.2 (so does apache 2.4.37 and prior) for http2 to work properly. I've had to do this with prior IUS http24u versions, however I think I just rebuilt the modules before. The openssl shipped with centos is only 1.0.1 and doesn't work with h2/http2 ... although apache 2.4.x still tries to send an h2 header the browser never actually upgrades the connection. With openssl1.0.2 it works wonderfully.

So all I have done is build an openssl package that sits by itself in /usr/local/openssl and built against it rather than stock. Stock can remain installed without issue. ld.conf needs an entry is all.

Well that was more long winded than I'd intended. Hope any of it helps.
